Add pain points for public sector organizations (#476)
publiccodenet/about: activities/value-and-impact/pain-points/public-organizations.md

---
type: Resource
---

# Public organizations maturing in their approach to digital public infrastructure

This resource contains an unordered and non-exhaustive list of painpoints that public organizations have shared with us during discussions about membership of the Foundation for Public Code and codebase stewardship.
This resource can be used to inform our user research, value proposition mapping and business case development for potential members, as well as provide a general overview of the context in which we operate.

These are common challenges faced by public organizations as they mature in their approaches to their digital public infrastructure.

## Operational delivery

Developing the exertise to operationally manage digital innovation projects, including in-house development, procurement, and the management of third party vendors, often without in-house technical expertise.

## Economies of scale and silos

> Learning to work together with partner-organizations to collaborate around a single shared system, rather than each having their own.

Reviewer: ```suggestion
Learning to work together with partner organizations to collaborate around a single shared system, rather than each having their own.
```

---
type: Resource
---

# Public organizations maturing in their approach to digital public infrastructure

This resource contains an unordered and non-exhaustive list of painpoints that public organizations have shared with us during discussions about membership of the Foundation for Public Code and codebase stewardship. 

> This resource can be used to inform our user research, value proposition mapping and business casing for potential members, as well as provide a general overview of the context in which we operate. 

Reviewer: ```suggestion
This resource can be used to inform our user research, value proposition mapping and business case development for potential members, as well as provide a general overview of the context in which we operate. 
```

> As public organizations mature in their approaches to their digital public infrastructure, typical challenges include: 

Reviewer: ```suggestion
These are common challenges faced by public organizations as they mature in their approaches to their digital public infrastructure.
```

No need for a colon as the subheadings aren't bullets.

## Operational delivery

Developing the exertise to operationally manage digital innovation projects, including in-house development, procurement, and the management of third party vendors, often without in-house technical expertise.

## Economies of scale and silos

Learning to work together with partner-organizations to collaborate around a single shared system, rather than each having their own.

## Sustainable relationships with third parties

> Avoiding lock-in situations, in which the organization's data and/or digital workflow is controlled by an third party, leading to a reduced autonomy and potentially prohibitive liscensing, upgrade and migration costs.

Reviewer: ```suggestion
Avoiding lock-in situations, in which the organization's data or digital workflow is controlled by a third party, leading to reduced autonomy and potentially prohibitive liscensing, upgrade and migration costs.
```

> ## Future-proofing

Reviewer: ```suggestion
## Futureproofing
```

> Understanding how decisions about their digital public infrastructure will open up or limit possibility of future upgrades, add-ons and innovation potential in the future.

Reviewer: ```suggestion
Understanding how decisions about their digital public infrastructure will open up or limit the possibility of future upgrades, add-ons and innovation potential.
```

## Technical debt

> Working around an ever increasing set of aging digital systems that create dependies and a potential drag on service delivery.

Reviewer: ```suggestion
Working around an ever increasing set of aging digital systems that create dependencies and a potential drag on service delivery.
```

## Balancing local and global

> Stiking a balance between existing offers of global parties, while still investing in local businesses.

Reviewer: ```suggestion
Striking a balance between existing offers by global parties, while still investing in local businesses.
```
